src/data/events.json

```diff
@@ -86,5 +86,16 @@
     "location": "Chennai",
     "communityName": "Women Tech Makers (WTM) Chennai",
     "communityLogo": "https://i.ibb.co/ymLgFRk3/images.png"
-  }
+  },
+  {
+  "eventName": "MongoDB User Group Chennai: April Meetup - The AI Awakens",
+  "eventDescription": "Join us for an exciting in-person meetup where we explore the power of AI and MongoDB! Whether you're a developer, data enthusiast, or just curious about the intersection of AI and databases, this is the perfect place to learn, network, and grow.",
+  "eventDate": "2024-04-19",
```

Comment:
😅 uhm I guess the event ended with this one, can you please update this @rishiagrawal2609

Reply:
Made it to 2025-04-19